developers/compute-to-data/README.md

```diff
@@ -20,19 +20,6 @@ The Ocean Protocol provides a compute environment that you can access at the fol
 
 We suggest reading these guides to get an understanding of how compute-to-data works:
 
-### Architecture & Overview Guides
-
-* [Architecture](compute-to-data-architecture.md)
-* [Datasets & Algorithms](compute-to-data-datasets-algorithms.md)
-* [Writing Algorithms](compute-to-data-algorithms.md)
-* [Compute options](compute-options.md)
-
-### User Guides
```

Comment:
These as well need to be updated, not deleted, maybe in #1513 since that PR contains the modifications for c2d.